tool/generator/lib/src/inputs/interfaces.dart

```diff
@@ -115,6 +115,7 @@ final windowsRuntimeTypesToGenerate = <String>{
   'Windows.Globalization.Calendar',
   'Windows.Globalization.PhoneNumberFormatting.PhoneNumberFormatter',
   'Windows.Globalization.PhoneNumberFormatting.PhoneNumberInfo',
+  'Windows.Graphics.Printing3D.Printing3DMultiplePropertyMaterial',
```

Comment:
Did you pick this one just as a test case? (It's fine -- just curious...)

Reply:
Yes! This seemed like a great choice as it is `Activatable` so I was able to test it very easily.